### PR TITLE
feat: support tolerations in helm values

### PR DESCRIPTION
**Pull request by royhadad**
https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/

### CODE REVIEW

File: charts/datree-admission-webhook/README.md [hunk 1]
```diff
@@ -40,28 +40,28 @@ For further information about Datree flags see [CLI arguments](https://hub.datre
 
 ### Parameters
 
-| Parameter                             | Description                                                                               | Default                                                                                                                                  |
```
Comment (royhadad) [inline charts/datree-admission-webhook/README.md:43]: just a reformat, no actual change

File: charts/datree-admission-webhook/values.yaml [hunk 2]
```diff
@@ -58,6 +58,7 @@ resources: {}
 #   memory: 48Mi
 nodeSelector: {}
 affinity: {}
+tolerations:
```
Comment (TzlilSwimmer123) [inline charts/datree-admission-webhook/values.yaml:61]: I see that the initial val suppose to be `tolerations: []` according to prometheus example:
https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus/values.yaml

File: charts/datree-admission-webhook/templates/deployment.yaml [hunk 3]
```diff
@@ -29,7 +29,10 @@ spec:
       {{- end }}
       {{- with .Values.affinity }}
       affinity: {{ toYaml . | nindent 8 }}
-        {{- end }}
+      {{- end }}
+      {{- if .Values.tolerations }}
```
Comment (royhadad) [inline charts/datree-admission-webhook/templates/deployment.yaml:33]: with?

Comment (myishay) [reply charts/datree-admission-webhook/templates/deployment.yaml:33]: doesn't really matter

**Review by myishay — APPROVED** [review]

**Pull request merged**